README.md

```diff
@@ -1,36 +1,128 @@
+| `B  ` | Meets all of:<ul><li>Reviewed current and new behavior with customer representative.</li><li>Change is <= 8 <abbr title="lines of code">LoC</abbr></li><li>Bug's original (buggy) behavior was captured in a unit test prior to this change.</li><li>Change includes 1 changed unit test, matching intended behavior alteration.</li></ul> |
+| `B!!` | Change includes unit tests for new behavior. |
+| `B**` | No automatic tests, or unfinished implementation. |
+
+### Refactoring or Remodeling
+
+A Refactoring or Remodeling intends to alter the program in some way without changing any behavior. The risk levels indicate the probability of the commit living up to that intention, based on how the code change was executed.
+
+**Known Risks**
+
+* May cause a bug.
+* May fix a bug.
+* May change a behavior in a way that doesn't impact a user.
+* May force a test update.
+
+| Code | Known Approaches |
+| --- | --- |
+| `r  ` | Provable refactoring<sup>[2]</sup> |
+| `R  ` | Test-supported Procedural Refactoring<sup>[3]</sup> |
+| `R!!` | Identified single, named refactoring, but executed by editing code or without whole-project test coverage. |
+| `R**` | Remodeled by editing code, even in small chunks. |
+
+### Documentation
+
+Changes that don't impact the code, but do change documentation around the code. Note that this does not include end-user documentation<sup>[1]</sup>.
+
+**Known Risks**
+
+* May mislead future developers.
+* May mislead other stakeholders.
+* May alter team processes in ways that have unintended consequences.
+
+| Code | Known Approaches |
+| --- | --- |
+| `d  ` | Developer-visible documentation, not in a source file, or verified to generate byte-identical compilation. |
+| `D  ` | Dev-impacting only, but changes compilation or process. E.g., changing text on a dev-only screen, or changes code-review checklist. |
+| `D!!` | Alters an important process. |
+| `D**` | Trying out a process change that is intended to gain info, not to work. |
+
+## Extension Intentions
+
+Each project can define a set of extension intentions. Each project should define which extension codes it uses. It is up to each project to define the approaches for each of the 4 risk levels.
+
+These are some common intentions, each used in several projects. Each also lists alternatives used in projects that don't use the code.
+
+| Prefix | Name | Intention | Alternatives |
+| --- | --- | --- | --- |
+| M | Merge | Merge branches | Use `F`, `B`, or `R`, based on the main intention of the branch, with risk level based on maximum for any individual commit in the branch. Optionally leave blank for merge from main to a feature branch. |
+| T | Test-only | Alter automated tests without altering functionality. May include code-generating code that just throws a `NotImplementedException` or similar approaches. | Use `f` or `b`, depending on which kind of work this test is going to validate. It is a lower-case letter unless you also change product code. |
```

Comment:
What do you think of using `r` when refactoring entirely within test code?

Reply:
I like it. I'll add that as another known way to be at that level, as well as an alternative to t.